### ISSUE TITLE
Getting training and validation accuracy for object detection

### ISSUE THREAD

**Issue by SandeepSreenivasan** [tensorflow/models#2320]
Can anyone tell me how to get the training and validation accuracy for objection detection model. When I run the train.py and eval.py scripts loss is getting printed but how to get the accuracy values?

**Comment by ArjanSchouten** [comment]
Better ask on Stackoverflow!

You can use tensorboard for that.
Run ```tensorboard --logdir=results```

Start train.py: ```python train.py --train_dir=results/checkpoints ...```
Start eval.py: ```python eval.py --eval_dir=results/eval ...```

**Comment by michaelisard** [comment]
Yes, stackoverflow is a better forum for this kind of question.

**Comment by atulapra** [comment]
@SandeepSreenivasan did you figure out how to check accuracy?
I'm facing the same issue.
